### ISSUE TITLE
Deploy Trailhead to All Desktop Browser Touchpoints

### ISSUE THREAD

**Issue by davismtl** [mozilla/fxa#1305]
On Firefox Desktop, we currently only display the trailhead flow from about:welcome and /whatsnew.

This specific issue is about making the trailhead flow visible from all the other entrypoints on the desktop browser. (account menu, hamburger menu, settings, send tab menu, etc)

Let's audit and make sure that the flow makes sense in all of those contexts.

**Comment by shane-tomlinson** [comment]
Is this a dup of #1245?

**Comment by shane-tomlinson** [comment]
> Is this a dup of #1245?

No, it's not, #1245 is a tracker.

**Comment by clouserw** [comment]
This is scheduled to have UX delivered around Aug 1 according to the Skyline schedule

**Comment by juliemccracken** [comment]
Added this to a post Trailhead epic to differentiate between this work and Skyline work. I repurposed the Skyline Epic to be the Epic for the FxA Toolbar Menu, and the created a release for Skyline that I'm adding the Skyline related epics to.

**Comment by clouserw** [comment]
We talked about this in triage today and we're not clear what is left to do here.  @ryanfeeley is this something you're working on?  @johngruen isn't sure there is work to do here

**Comment by davismtl** [comment]
Perhaps I can clarify.

In June, we made changes to our about:welcome and /whatsnew flows for Trailhead. More specifically, we made flow changes if `?style=trailhead` was specified. This is because we didn't have time to consider the repercussions across all of our touchpoints. (e.g. menus, preferences, etc)

The desired outcome of this GitHub issue is that all of our entrypoints/touchpoints to login/register within the browser are aligned with Trailhead. In fact, we might say that it would deprecate `style=trailhead` since it would be somewhat the global norm.

More specifically, all of our touchpoints should make some mention of the other services that users can access with their Firefox Account.

What work needs to be done? High level:

- Audit the other browser touchpoints to see if any changes are actually needed
- If so, determine if the flows that we rolled out for Trailhead would work globally or if UX changes might need to be made.
- Work on any identified changes and deploy

**Comment by clouserw** [comment]
Leaving this marked as `needs:ux` for the first two bullet points mentioned above.

**Comment by ryanfeeley** [comment]
@davismtl  I agree that we should get rid of using style=trailhead and standardize our flows.

Here is the desired standardized experience:
1. Ensure that the ecosystem promo appears on any email-first form (few see this page as they will encounter the ecosystem from the relying website when they enter their email there)
1. Remove the ecosystem promo from our password creation and entry forms (cluttering up this page affects conversion of these very important pages)
1. Return the subheading "to continue to {Relying Service}" (e.g. Firefox Sync) to every single page it was removed from in Trailhead.

Make sense?
